config/src/bloop/config/Config.scala

```diff
@@ -2,6 +2,7 @@ package bloop.config
 
 import bloop.config.PlatformFiles.Path
 import bloop.config.PlatformFiles.emptyPath
+import scala.annotation.unroll
```

Comment:
Ach, and you also need to run organize imports

Reply:
Remember to update docs, forget to run scalafix...